opentelemetry-sdk/tests/metrics/test_aggregation.py

```diff
@@ -114,13 +115,17 @@ def test_collect_delta(self):
         )
 
         synchronous_sum_aggregation.aggregate(measurement(1))
-        first_sum = synchronous_sum_aggregation.collect()
+        # _time_ns used here directly to simulate the instant the first
+        # collection process starts.
+        first_sum = synchronous_sum_aggregation.collect(_time_ns())
```

Comment:
nit could also use `1` and `2` instead of _time_ns() to make the tests more repeateable

Reply:
Right, fixed.